### ISSUE TITLE
CWS fine-tuning

### ISSUE THREAD

**Issue by Graf-D** [fastnlp/fastHan#23]
Is it possible to fine-tune fastHan model for the CWS task? The most preferable option would be a class method (something like `model.fine_tune(new_sentences, correct_segmentation)`). Loading the model via pytorch and training it or any other options would be also OK.

If that's possible, in what way exactly? How does the input look like? (e.g. "csv-file with two columns: 1. sentence 2. expected list of tokens") Does it depend on the dataset used for original training? (e.g. for fine-tuning model trained on PKU I should use an input format A, while for fine-tuning WTB I should use B)

If not, are you planning to add such an option?

**Comment by fdugzc** [comment]
there is no convenient interface for fine-tuning now. Technically, you can fine-tune the model by loading the model parameters and optimize the loss function manually. However, fine-tuning on one task may have negative effect on other tasks which I didn't experiment yet. 
I will add a more convenient interface if there is an amount of demand (but not in the next two months beacuse of personal matters)

**Comment by Graf-D** [comment]
Thank you for answering so quickly!
Can I use a model structure code and pre-trained weights matrix to make a fine-tuning by myself? I mean that I'd like to load weights and after that do more training via pytorch. If it's possible, what is an input format to train the model?
It looks like PKU, CNC and CTB datasets are different but was used in the (structurally) same model. If we want to train each of them, will the datasets be in the same format?

**Comment by fdugzc** [comment]
sorry that I forgot to reply your latest comment. Now fastHan 1.7 has the finetune feature, welcome to try.
